package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "zustand",
   "private": true,
-  "version": "4.3.2",
+  "version": "4.3.3",
```

Please don't change version.
```suggestion
  "version": "4.3.2",
```

---

Undone.